integration/deployment/Dockerfile

```diff
@@ -0,0 +1,7 @@
+#target: danm-deployer
+FROM alpine:3.8
+MAINTAINER Nandor Galambosi <nandor.galambosi@nokia-bell-labs.com>
+COPY .cache/kubectl /opt/bin/
+COPY bin/* /opt/bin/
```

Comment:
internally we have discussed installing the CNI binaries of all backends, which we support with the dynamic level
These are (in addition to danm):
- macvlan
- sriov (specific version for the time being

Source from where you can get these:
- macvlan: https://github.com/containernetworking/plugins/releases/download/v0.7.0/cni-plugins-amd64-v0.7.0.tgz  (version could be a config parameter ofc)
- sriov: this one we build within our product, but here is the code for reference:
&&   mkdir -p ${GOPATH}/src/github.com/intel/sriov-cni \
&&  curl -fsSL -k https://github.com/intel/sriov-cni/archive/v0.2.tar.gz | tar zx --strip-components=1 -C ${GOPATH}/src/github.com/intel/sriov-cni \
&&  cd ${GOPATH}/src/github.com/intel/sriov-cni \
&&  ./build \
&&  mv ${GOPATH}/src/github.com/intel/sriov-cni/bin/sriov /opt/cni/bin/sriov \
right now we need the specific, 0.2, or latest 0.4 version

+1: if you decide to go the "fully integrated Flannel" way, you can get its CNI from here: 
go get -v github.com/containernetworking/plugins/plugins/meta/flannel